src/current/v24.3/stream-a-changefeed-to-amazon-msk.md

```diff
@@ -0,0 +1,340 @@
+    {"after": {"address": "87351 David Ferry Suite 24", "city": "amsterdam", "credit_card": "7689751883", "id": "58f66df9-e2ef-48bf-bdbe-436e8caa0fae", "name": "Grant Murray"}}
+    {"after": {"address": "35991 Tran Flats", "city": "amsterdam", "credit_card": "6759782818", "id": "6e8d430d-9a3b-4519-b7ab-987d21043f6a", "name": "Mr. Alan Powers"}}
+    {"after": {"address": "65320 Emily Ports", "city": "amsterdam", "credit_card": "7424361516", "id": "74e8c91b-9534-4e40-9d19-f23e14d24114", "name": "Michele Grant"}}
+    {"after": {"address": "85363 Gary Square Apt. 39", "city": "amsterdam", "credit_card": "0267354734", "id": "99d2c816-2216-40f3-b60c-f19bc3e9f455", "name": "Mrs. Wendy Miller"}}
+    {"after": {"address": "68605 Shane Shores Suite 22", "city": "amsterdam", "credit_card": "5913104602", "id": "ae147ae1-47ae-4800-8000-000000000022", "name": "Crystal Sullivan"}}
+    {"after": {"address": "41110 Derrick Walk Apt. 42", "city": "amsterdam", "credit_card": "2866469885", "id": "b3333333-3333-4000-8000-000000000023", "name": "Michael Lewis"}}
+    {"after": {"address": "47781 Robinson Villages Apt. 41", "city": "amsterdam", "credit_card": "6596967781", "id": "b40e7c51-7e68-43f1-a4df-92fa8a05c961", "name": "Abigail Sellers"}}
+    ...
+    {"resolved":"1725982589714395510.0000000000"}
+    ...
+    ~~~
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="scram-setup-steps">
+
+## Step 1. Create an MSK cluster with SCRAM authentication
+
+1. In the AWS Management Console, go to the [Amazon MSK console](https://console.aws.amazon.com/msk/home) and click **Create cluster**.
+1. Select **Custom create**, name the cluster, and select **Provisioned** as the cluster type. Click **Next**.
+1. Select the VPC for the MSK cluster with the subnets and security group. The VPC selection is important because the MSK cluster must be in the same VPC as the CockroachDB instance and Kafka client machine. Click **Next**.
+1. Under **Access control methods** select **SASL/SCRAM authentication**. Click **Next**.
+1. Continue to select the required configuration options for your cluster. Click **Next**.
+1. Review the cluster details, and then click **Create cluster**.
+1. Once the cluster is running, click **View client information** in the **Cluster summary** box. Copy the endpoint addresses, which will be similar to `b-1.msk-cluster_name.1a2b3c.c4.kafka.us-east-1.amazonaws.com:9096,b-2.msk-cluster_name.1a2b3c.c4.kafka.us-east-1.amazonaws.com:9096,b-3.msk-cluster_name.1a2b3c.c4.kafka.us-east-1.amazonaws.com:9096`. Click **Done** to return to the cluster's overview page.
+
+## Step 2. Store the SCRAM credentials
+
+In this step, you'll store the SCRAM credentials in [AWS Secrets Manager](https://docs.aws.amazon.com/secretsmanager/) and then associate the secret with the MSK cluster.
```

Comment:
first mention of 'SCRAM' could be a link to our own SCRAM docs: https://www.cockroachlabs.com/docs/v24.3/security-reference/scram-authentication.html